code/gui/scripteditor.cpp

```diff
@@ -0,0 +1,123 @@
+#include <QFileDialog>
+#include <QTextStream>
+
+#include "scripteditor.h"
+#include "ui_scripteditor.h"
+
+#include "../utility/logger.h"
+
+ScriptEditor::ScriptEditor(QWidget *parent) :
+        QDialog(parent),
+        m_activeFile(),
+        ui(new Ui::ScriptEditor) {
+    ui->setupUi(this);
+
+    // Make the font look like code
+    QFont font;
+    font.setFamily("Courier");
+    font.setStyleHint(QFont::Monospace);
+    font.setFixedPitch(true);
+    font.setPointSize(8);
+    QFontMetrics metrics(font);
+    const int tabSize = 4;
+    const int tabStopWidth = tabSize * metrics.width(' ');
```

Comment:
again, `variable_name`, and `functionName`, just to stay consistent with the style guide.